.github/workflows/elementor-tests.yml

```diff
@@ -4,6 +4,7 @@ on: [push, pull_request_target]
 
 jobs:
   run:
+    if: github.repository_owner == 'elementor'
```

Comment:
did you checked if it working with `pull_request_target` event?

Reply:
Yes